### ISSUE TITLE
coffee-newline-and-indent modifies long lines

### ISSUE THREAD

**Issue by taylorzane** [defunkt/coffee-mode#341]
When pressing <kbd>return</kbd>/<kbd>enter</kbd> on a long line, the line is automatically modified to some pseudo-unpredictable format.

Example:
```coffee-script
funReallyLongFunctionNameThatShouldBreakCoffeeMode = (a, b, c, d, e, f, g, h, u, j, k, l) ->
```

With cursor at the end of the line (`C-e`), pressing <kbd>return</kbd>/<kbd>enter</kbd> yields this:

```coffee-script
funReallyLongFunctionNameThatShouldBreakCoffeeMode = (a, b, c, d, e,
f, g, h, u, j, k, l) ->
```

I'm unsure what the process is behind determining what line needs highlighting and what doesn't.

I would just like an easy way to disable this feature/bug while still being able to use the "newline then indent" feature.

**Comment by syohex** [comment]
I have fixed at #342. Please check latest version.

**Comment by taylorzane** [comment]
Thanks @syohex! That worked perfectly.
